src/sagemaker_inference/transformer.py

```diff
@@ -111,32 +111,38 @@ def transform(self, data, context):
             model_dir = properties.get("model_dir")
             self.validate_and_initialize(model_dir=model_dir)
 
-            input_data = data[0].get("body")
+            response_list = []
 
-            request_processor = context.request_processor[0]
+            for _ in range(len(data)):
+                input_data = data[0].get("body")
```

Comment:
shouldn't this be data[i] ?

Reply:
Thanks, that is correct, this was a wrong commit for the PR. Corrected now.